server/meta.jsx

```diff
@@ -192,6 +192,17 @@ function getSDKScriptAttributes(sdkUrl : ?string, allAttrs : ?{ [string] : strin
     return attrs;
 }
 
+function clearQueryFromLegacySDKUrl(sdkUrl : string ) : string {
+    const { protocol, hostname, port, pathname } = urlLib.parse(sdkUrl);
+
+    return isLegacySDKUrl(String(hostname), String(pathname)) ? urlLib.format({
```

Comment:
I don't think we need the `String()` conversion. We don't do that with the other usage of `urlLib.parse()`
```suggestion
    return isLegacySDKUrl(hostname, pathname) ? urlLib.format({
```